python/scope.py

```diff
@@ -949,7 +957,10 @@ async def eval_expr(self, expr):
                 # Support old syntax
                 rel_file_path = expr.children[0].value + ".n"
             file_path = os.path.join(os.path.dirname(self.file_path), rel_file_path)
+            with open(self.file_path, "r", encoding="utf-8") as f:
+                self.stack_trace.append((expr, File(f, name=os.path.relpath(self.file_path, start=self.base_path))))
             val = await eval_file(file_path, self.base_path)
+            self.stack_trace = self.stack_trace.__add__(val.stack_trace[:])
```

Comment:
Why not do `self.stack_trace + val.stack_trace`?

Reply:
Or better yet,
```suggestion
            self.stack_trace += val.stack_trace
```

Reply:
Added